core/src/components/buttons/buttons.tsx

```diff
@@ -1,13 +1,17 @@
 import type { ComponentInterface } from '@stencil/core';
 import { Component, Host, Prop, h } from '@stencil/core';
 
-import { getIonMode } from '../../global/ionic-global';
+import { getIonTheme } from '../../global/ionic-global';
 
+/**
+ * @virtualProp {"ios" | "md" | "ionic"} theme - The visual appearance of the component.
```

Comment:
Same answer as: https://github.com/ionic-team/ionic-framework/pull/29132#discussion_r1520488732

However we _could_ list the virtual prop since the logic crawls to the nearest element with a `mode` attribute.